en/components/general/update_guide.md

```diff
@@ -44,6 +44,47 @@ Unfortunately not all changes can be automatically updated. Changes bellow are s
 
 For example: if you are updating from version 6.2.4 to 7.1.0 you'd start from the "From 6.x .." section apply those changes and work your way up:
 
+### From 8.1.x to 8.2.x
```

Comment:
@skrustev I think you should add a section about `ghostImageClass` and how it should be migrated manually.